src/status_im2/common/floating_button_page/floating_container/view.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im2.common.floating-button-page.floating-container.view
+  (:require [quo.foundations.colors :as colors]
+            [quo.theme :as quo.theme]
+            [react-native.blur :as blur]
+            [react-native.core :as rn]
+            [react-native.platform :as platform]
+            [react-native.reanimated :as reanimated]
+            [react-native.safe-area :as safe-area]
+            [status-im2.common.floating-button-page.floating-container.style :as style]))
+
+(def duration 100)
+
+(defn blur-container-props
+  [theme]
+  {:blur-amount      12
+   :blur-radius      25
+   :blur-type        (quo.theme/theme-value :light :dark theme)
+   :style            (when platform/ios?
+                       {:width              "100%"
+                        :padding-horizontal 20
+                        :padding-vertical   12})
+   :background-color (colors/theme-colors colors/white-70-blur colors/neutral-80-opa-1-blur theme)})
+
+(defn- blur-container
+  [props & children]
+  [rn/view
+   (merge props
+          {:width    "100%"
+           :overflow :hidden}
+          (when platform/android?
+            {:padding-horizontal 20
+             :padding-vertical   12}))
+   (into [blur/view (blur-container-props (:theme props))] children)])
+
+"
+-        on-layout : will trigger to dynamically get the height of the container to screen using it.
+- show-background? : blurred container that is activated when this component is on top of the page content.
+-  keyboard-shown? : keyboard  is visible on the current page.
+"
+(defn get-margin-bottom
+  [show-background? keyboard-will-show?]
+  (if platform/android?
+    0
+    (cond keyboard-will-show?                        (safe-area/get-top)
+          (and show-background? keyboard-will-show?) (safe-area/get-bottom)
+          :else                                      (safe-area/get-bottom))))
```

Comment:
`cond` can be `(if keyboard-will-show?     (safe-area/get-top)   (safe-area/get-bottom)`

Reply:
will adjust!